### PR TITLE
Interview Questions update

### PR DESCRIPTION
**Pull request by vixg23**
Updated interview Questions. Pull Request test.

Screen Shot 2022-08-29 at 6 37 28 PM

### CODE REVIEW

File: interviewPractice.txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+commit & push to git
+Code Review (Pull request)
+Approve and merge to master
+
+(
+    git commands
+    ------------
+
+    git branch
+    git fetch --all
```
Comment (avimech4u) [inline interviewPractice.txt:34]: Null check is missing

**Review by avimech4u — CHANGES_REQUESTED** [review]
Left some comments please

File: interviewPractice.txt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Day to day activites - Develop webpages from wireframes, build components for webpages, Fetch and update  data form/to the server
+Web development Technologies used - React, Redux, Javascript(ES6), SCSS for styling, HTML5.
+For project management - JIRA, Github
+
+Previous project description - Health care
+Web development Technologies used - HTML, CSS, JQuery
+
+
+2. Team Coding Process
+-----------------------
```
Comment (pamarthyKusuma) [inline interviewPractice.txt:20]: null check point

**Review by pamarthyKusuma — CHANGES_REQUESTED** [review]
request change

**Review by avimech4u — APPROVED** [review]

**Review by pamarthyKusuma — APPROVED** [review]

**Pull request merged**